Documentation/OSMPrivatePreview/ReadMe.md

```diff
@@ -30,6 +32,25 @@ osm metrics enable --namespace "test1, test2"
          * Example: `kubectl apply -f container-azm-ms-agentconfig.yaml`
 4. The configuration change can take upto 15 mins to finish before taking effect, and all omsagent pods in the cluster will restart. The restart is a rolling restart for all omsagent pods, not all restart at the same time.
 
+### Azure Arc for Kuberentes
+This section assumes that you already have your kubernetes distribution connected via Azure Arc. If not learn more [here.](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster)
+
+1. Install Arc enabled Open Service mesh on your Arc cluster. Learn more [here](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/tutorial-arc-enabled-osm#navigating-the-osm-dashboard)
+2. Install Azure Monitor Container Insights on Arc. If not installed already. Learn more how to install [here](https://docs.microsoft.com/azure/azure-monitor/containers/container-insights-enable-arc-enabled-clusters)
+3. Configure OSM to allow Prometheus scraping, follow steps from [here](https://docs.microsoft.com/en-us/azure/aks/servicemesh-osm-about?pivots=client-operating-system-linux#configure-osm-to-allow-prometheus-scraping)
+3.  To enable namespace(s), download the osm client library [here](https://docs.microsoft.com/en-us/azure/aks/servicemesh-osm-about?pivots=client-operating-system-linux#osm-service-quotas-and-limits-preview) & then enable metrics on namespaces
+```bash
+# With osm
+osm metrics enable --namespace test
+osm metrics enable --namespace "test1, test2"
+
+```
+4. On your Azure Monitor Container Insights for Arc.
+ * Download the configmap from [here](https://github.com/microsoft/Docker-Provider/blob/ci_prod/kubernetes/container-azm-ms-osmconfig.yaml)
+     * Add the namespaces you want to monitor in configmap `monitor_namespaces = ["namespace1", "namespace2"]`
+     * Run the following kubectl command: kubectl apply -f<configmap_yaml_file.yaml>
+         * Example: `kubectl apply -f container-azm-ms-agentconfig.yaml`
```

Comment:
Could we add the right osm configmap in the example? That way it will be easier to copy.

Reply:
Updated